docs/usage.md

```diff
@@ -43,9 +43,9 @@ and safe to directly insert variable data via f-strings:
 
 ### Conditional Rendering
 
-`None` will not render anything. This can be useful to conditionally render some content.
+`None` and `False` will not render anything. This can be useful to conditionally render some content.
```

Comment:
Actually, `None`, `False`and `True`will not render anything.
Although handling `False` value is useful for the `and` syntax like in `is_allowed and div["Access granted"]`, the `True` value can be used for the `or` syntax like `is_allowed or div["Access denied"]` (which is the equivalent to `not is_allowed and div["Access denied"]`).

Reply:
Good catch, explicitly calling out or and showing that True does not render anything is a good idea, I have added it now!